index.js

```diff
@@ -43,6 +44,14 @@ if (enableMetrics) {
   })
 }
 
+app.get('/health', (req, res) => {
+  if (client.ping()) {
+    res.status(200).end()
+  } else {
+    res.status(503).end()
+  }
+})
```

Comment:
Yea, this should probably just go in the `bootApp` method since that's where other routes/handlers live.

@izakp I also frequently see the _first_ request or something sometimes time out or have a `Cannot get /` message as you point out. I think it's just some development-only issue (but have not investigated and am sort of speculating), I can't imagine it's an issue in production. Are you imagining ELB hitting a health check endpoint as the instance has just booted and getting a false alert (and taking it out of rotation)?

Reply:
Hrmm - if I place the route handler in the `bootApp` method it seems to be overridden (and redirects to the GraphiQL page)

Reply:
It works as expected in context of the `bootApp` function though